spec/task-spec.js

```diff
@@ -0,0 +1,53 @@
+import Task from '../src/task.js'
+import {expect} from 'chai'
+
+class SampleClass extends Task {
+
+}
+
+class ImplementedTask extends Task {
+  run() {
+    // Something
+  }
+  done() {
+    // Job done!
+  }
+}
+
+describe('Task', () => {
+
+  it('should trigger an error when instantiating a new Task', () => {
+    const fn = () => new Task()
+    expect(fn).to.throw(Error)
+    expect(fn).to.throw('Can not construct Task class')
```

Comment:
That's the purpose of testing, the message should not change and if it does, the test should be updated too.